### PR TITLE
Fix codecov and speed up tests

### LINKED ISSUE #632: Codecov is broken
tl;dr misconfigured after the merge

https://codecov.io/gh/arrow-kt/arrow/commit/b7f952bb65054d86370ca5aaea08cc59a93da6ba

### PR DESCRIPTION
**Pull request by jereksel**
Fixes #632

### CODE REVIEW

File: modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt [hunk 1]
```diff
@@ -5,8 +5,7 @@ import arrow.core.Option
 import arrow.typeclasses.Const
 import arrow.typeclasses.Monoid
 
-@PublishedApi
-internal object AndMonoid : Monoid<Boolean> {
+object AndMonoid : Monoid<Boolean> {
```
Comment (nomisRev) [inline modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt]: Why is this not internal anymore?

Comment (jereksel) [reply modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt]: I guess I was wrong - IDE shows error, but it compiles fine.

Comment (pakoito) [reply modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt]: Never seen that happen before hahaha

Comment (jereksel) [reply modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt]: https://youtrack.jetbrains.com/issue/KT-23668 - it's propably that

**Comment by jereksel** [comment]
Could you merge this? I'm working on ap tests rewrite and change in coverage would be very useful.

**Pull request merged**